autoconsent/autoconsent-impl/src/main/res/values/strings-autoconsent.xml

```diff
@@ -15,7 +14,8 @@
   ~ limitations under the License.
   -->
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.duckduckgo.autoconsent.api">
-
-</manifest>
\ No newline at end of file
+<resources>
+    <string name="autoconsentTitle">Manage Cookie Pop-ups</string>
+    <string name="autoconsentDescription">When DuckDuckGo detects cookie consent pop-ups on sites you visit, we can automatically set your cookie preferences to minimize cookies and maximize privacy, then close the pop-ups.</string>
+    <string name="autoconsentToggle">Let DuckDuckGo manage cookie consent pop-ups</string>
+</resources>
```

Comment:
I guess these will be translated in separate PR (?)

Reply:
Yup